model/database/metrics.yaml

```diff
@@ -133,3 +134,12 @@ groups:
     attributes:
       - ref: db.client.connection.pool.name
         requirement_level: required
+
+  - id: metric.db.response.item_count
```

Comment:
discussed naming at the OTel DB Semconv meeting, suggesting to change the metric  name to 
`db.client.response.row_count`. It would also make sense to add a similar span attribute -  `db.response.row_count` if you don't need it for cosmos.

full notes https://docs.google.com/document/d/1zTi_Z4WBisytPnXTpwtnGOK-4Gv5oSTuMHvIQ8--5hg/edit?tab=t.0